PayloadPkg/Include/Library/PayloadLib.h

```diff
@@ -56,6 +56,7 @@ typedef struct {
   UINT32        Version;
   UINT32        HeapSize;
   UINT32        HeapAddr;
+  UINT32        HobListPtr;
```

Comment:
Can you move this after OS State variable? That way its backwards compatible.

Reply:
Moved to the end as requested.